### PR TITLE
Inherit the Artist.draw docstring in subclasses.

### PR DESCRIPTION
**Pull request by anntzer**
## PR Summary

Per https://github.com/matplotlib/matplotlib/pull/16004#discussion_r360711135.

## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: lib/matplotlib/axes/_base.py [hunk 1]
```diff
@@ -2613,7 +2613,7 @@ def _update_title_position(self, renderer):
     # Drawing
     @martist.allow_rasterization
     def draw(self, renderer=None, inframe=False):
```
Comment (dopplershift) [inline lib/matplotlib/axes/_base.py]: What about the parameters here? They're undocumented (though they were before too).

**Comment by dopplershift** [comment]
The 3 different signatures makes my skin crawl. By virtue of `Artist.draw` taking `*args*` and `**kwargs` it was clearly meant for subclasses to take additional parameters. I think the ones that abandon this with `draw(self, renderer)` bother me the most because they're more restrictive.

Actually, technically they're *all* more restrictive than their base class...

**Comment by anntzer** [comment]
I would rather just always make the signature `draw(self, renderer)` and figure out whether we can deprecate the `inframe` kwarg to Figure.draw (just one special case doesn't seem sufficient to make a more complex signature worth it).

**Comment by timhoffm** [comment]
At least the base class should not have any arguments except renderer. From a design perspective, it's perfectly fine for subclasses to add additional optional arguments. No need to anticipate that with `*args, **kwargs`. Removing this needs deprecation but is straight forward. Looking into `inframe` may or may not be done simultaneously.

Anyway, that should be done in a separate PR.

**Pull request merged**